src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,85 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String NAME_SUPPLY = "supply";
+    private static final String NAME_BUY = "buy";
+    private static final int INDEX_BUY = 0;
```

Comment:
```suggestion
    private static final int BUY_INDEX = 0;
```
the same for the rest